app/gateway/2.8.x/plan-and-deploy/security/secrets-management/backends/env.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Environment Variable Vault
+beta: true
+badge: free
+---
+
+## Configuration
+
+Storing secrets in Environment Variables is a common way as they can be injected at build time.
+There is no prior configuration needed.
```

Comment:
There is `KONG_VAULT_ENV_PREFIX` which can be given in ? ´kong.conf`:
```
vault_env_prefix=SECRET_
```

or as environment variable:
```
KONG_VAULT_ENV_PREFIX=SECRET_
```

Reply:
Should we refrain from suggesting kong.conf entries as we may change the naming scheme. This will be uncomfortable to migrate away from (think deprecation phases etc).